### PR TITLE
Feat/UI update

### PR DESCRIPTION
**Pull request by sohanemon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "plaiceholder": "^3.0.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "recharts": "^2.13.3",
```
Comment (Hanggi) [inline package.json:27]: This is not used, please remove it.

File: public/assets/images/派页.docx
Comment (Hanggi) [inline public/assets/images/派页.docx:1]: Don't put chinese named file into repo, you can create a 'doc' dir and put it there

File: app/model.tsx
Comment (Hanggi) [inline app/model.tsx:1]: This file is not used, please delete it.

File: public/assets/images/派页.docx
Comment (sohanemon) [reply public/assets/images/派页.docx:1]: done

File: app/model.tsx
Comment (sohanemon) [reply app/model.tsx:1]: done

**Pull request merged**